### ISSUE TITLE
order of members in sourcefiles

### ISSUE THREAD

**Issue by PtrMan** [opennars/opennars#173]
The order of the members and methods should be like this for a good readability:
* public members
* private members
* public methods
* private methods
* subclasses

For the following classes with a high priority
* NAR

**Comment by TonyLo1** [comment]
More detail required here to understand the issue.

**Comment by freemo** [comment]
@TonyLo1 he is refering to the order members of a class should appear in a class file. There is a standard suggested ordering which is he is attempting to elucidate, though in fact its more detailed that this. But I agree we should implement the standrd. This would be duplicated by #102

**Comment by TonyLo1** [comment]
Yes, he added them after my comment. Prior to that it was just a title :)
